modules/webm/resource_importer_webm.cpp

```diff
@@ -0,0 +1,94 @@
+/*************************************************************************/
+/*  resource_importer_ogg_vorbis.cpp                                     */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                    http://www.godotengine.org                         */
+/*************************************************************************/
+/* Copyright (c) 2007-2017 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2017 Godot Engine contributors (cf. AUTHORS.md)    */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+#include "resource_importer_webm.h"
+#include "video_stream_webm.h"
```

Comment:
Nitpicking, but the convention is to put one empty line between the main header `resource_importer_webm.h` and the other Godot ones. Headers should also be ordered alphabetically within a block. And same thing if you have system headers which should also be separate, so:
```
#include "resource_importer_webm.h"

#include "core/io/resource_saver.h"
#include "core/os/file_access.h"
#include "scene/resources/texture.h"
#include "video_stream_webm.h"

// if it were relevant, which it isn't here
#include <iostream>
```

Reply:
In your example, you used "core/io/resource_saver.h" instead of "io/resource_saver.h", but I spotted other modules using "io/resource_saver.h" style. Which is the one I should use here?

Reply:
You can use both, I want to enforce the usage of the `core/` prefix and remove it from the include path, but that will be done all at once on all includes, so it doesn't really matter which one you use now.

Reply:
Got it. I prefer that "core/" prefix too. For now, I make it align with other modules though. Thanks.